tests_end_to_end/tests/utils.py

```diff
@@ -0,0 +1,26 @@
+TEST_ITEMS = [
```

Comment:
In the future, probably better that your tests work with random data.

Reply:
this is planned as an improvement for the future once we get enough coverage for this first automation effort